frontend/apps/battleship-zk/.env.example

```diff
@@ -0,0 +1,8 @@
+VITE_NODE_ADDRESS=
+VITE_CONTRACT_ADDRESS=
+VITE_GASLESS_BACKEND_ADDRESS=
```

Comment:
Add VITE_ZK_PROOF_BACKEND_ADDRESS here please

Reply:
Added it in version with multiplayer